testing/test_mark.py

```diff
@@ -263,42 +270,44 @@ def test_hello():
         result = testdir.runpytest(p)
         result.stdout.fnmatch_lines(["*1 passed*"])
 
-    def test_mark_per_module(self, testdir):
+    @parametrize_mark_names
+    def test_mark_per_module(self, testdir, var_name):
         item = testdir.getitem("""
             import pytest
-            pytestmark = pytest.mark.hello
+            %s = pytest.mark.hello
             def test_func():
                 pass
-        """)
+        """ % var_name)
         keywords = item.keywords
         assert 'hello' in keywords
 
-    def test_marklist_per_class(self, testdir):
+    @parametrize_mark_names
+    def test_marklist_per_class(self, testdir, var_name):
         item = testdir.getitem("""
             import pytest
             class TestClass:
-                pytestmark = [pytest.mark.hello, pytest.mark.world]
+                %s = [pytest.mark.hello, pytest.mark.world]
                 def test_func(self):
                     assert TestClass.test_func.hello
                     assert TestClass.test_func.world
-        """)
+        """ % var_name)
         keywords = item.keywords
         assert 'hello' in keywords
 
-    def test_marklist_per_module(self, testdir):
+    @parametrize_mark_names
+    def test_marklist_per_module(self, testdir, var_name):
         item = testdir.getitem("""
             import pytest
-            pytestmark = [pytest.mark.hello, pytest.mark.world]
+            %s = [pytest.mark.hello, pytest.mark.world]
             class TestClass:
                 def test_func(self):
                     assert TestClass.test_func.hello
                     assert TestClass.test_func.world
-        """)
+        """ % var_name)
         keywords = item.keywords
         assert 'hello' in keywords
         assert 'world' in keywords
 
-    @pytest.mark.skipif("sys.version_info < (2,6)")
```

Comment:
Yes, `pytest` has dropped Python 2.5 in release `2.5.0`. From the `CHANGELOG`:

> dropped python2.5 from automated release testing of pytest itself which means it's probably going to break soon (but still works with this release we believe). 

So I decided to remove the line noise. :blush: